avocado/core/runners/utils/messages.py

```diff
@@ -202,6 +202,7 @@ def start_logging(config, queue):
     log.setLevel(log_level)
```

Comment:
I understand being grateful :smile:  but I guess this commit message was intended for a GitHub comment instead of a commit message... ?